swarm/api/http/server.go

```diff
@@ -99,7 +99,7 @@ func handler(w http.ResponseWriter, r *http.Request, a *api.Api) {
 				"[BZZ] Swarm: Protocol error in request `%s`.",
 				uri,
 			)
-			http.Error(w, "BZZ protocol error", http.StatusBadRequest)
+			http.Error(w, "BZZ protocol error. Specify protocol bzz:/, bzzr:/ or bzzi:/", http.StatusBadRequest)
```

Comment:
Minor nitpick: can we find a slightly better wording here? Looking at "Specify protocol" I wouldn't know what
to change with some guesswork. 

Maybe "Invalid request URL: need access protocol (bzz:, bzzr:, bzzi:) as first path component."
Please remove "BZZ protocol error".